.travis.yml

```diff
@@ -1,30 +1,289 @@
-language: R
-cache: packages
-warnings_are_errors: true
 
+# every build needs java8 so it can build and run Elasticsearch
 addons:
   apt:
     packages:
       - oracle-java8-set-default
 
-env:
-  matrix:
-  - ES_VERSION=1.0.0
-  - ES_VERSION=1.4.4
-  - ES_VERSION=1.7.2
-  - ES_VERSION=2.0.2
-  - ES_VERSION=2.1.2
-  - ES_VERSION=2.2.2
-  - ES_VERSION=2.3.5
-  - ES_VERSION=5.0.2
-  - ES_VERSION=5.3.3
-  - ES_VERSION=5.4.3
-  - ES_VERSION=5.6.9
-  - ES_VERSION=6.0.1
-  - ES_VERSION=6.1.4
-  - ES_VERSION=6.2.4
+# Manually specifying each build configuration.
+# Would be better to figure out how to build a matrix with two
+# languages but share the ES_VERSION matrix across all builds.
```

Comment:
Wow this does suck. Oh well.

Reply:
yeah idk what else to do. Once you need to do something as custom what we need, the limitations of Travis become clear.